src/test/java/teammates/test/cases/ui/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -135,16 +135,22 @@ private void testSubmitAction() throws Exception {
         submitPage = loginToStudentFeedbackSubmitPage("Alice", "Open Session");
 
         submitPage.fillResponseTextBox(1, 0, "Test Self Feedback");
+        assertEquals("18", submitPage.getResponseTextBoxLengthLabelValue(1, 0));
         submitPage.selectRecipient(2, 0, "Benny Charles");
         submitPage.fillResponseTextBox(2, 0, "Response to Benny.");
+        assertEquals("18", submitPage.getResponseTextBoxLengthLabelValue(2, 0));
         submitPage.selectRecipient(2, 1, "Drop out");
         submitPage.fillResponseTextBox(2, 1, "Response to student who is going to drop out.");
+        assertEquals("45", submitPage.getResponseTextBoxLengthLabelValue(2, 1));
```

Comment:
You should define String local variables for the ones that are also tested for its length. Let's avoid duplicating the String literals if possible, although we should ignore the duplicated literals already in the code.